core-bundle/src/Security/Voter/DataContainer/ArticleContentVoter.php

```diff
@@ -70,8 +70,13 @@ protected function hasAccessToRecord(TokenInterface $token, string $table, int $
     private function getPage(int $articleId): array|null
     {
         if (!\array_key_exists($articleId, $this->pageMap)) {
+            $this->pageMap[$articleId] = null;
```

Comment:
With this L26 needs to be charged to `@var array<int, array{id: int, type: string}|null>`.